asapdiscovery-ml/asapdiscovery/ml/trainer.py

```diff
@@ -124,6 +126,11 @@ class Trainer(BaseModel):
         None, description="Any extra config options to log to W&B."
     )
 
+    # artifact tracking options
+    upload_to_s3: bool = Field(False, description="Upload artifacts to S3.")
+    s3_settings: S3Settings | None = Field(None, description="S3 settings.")
```

Comment:
I didn't see this arg in the CLI stuff, do we want to be able to specify this there?

Reply:
sorry, I meant for the `s3_settings` specifically

Reply:
Ahhhhhhh, I would say lets leave this out of CLI as its composed of 3 separate (sensitive) environment variables, better to store in an env file,  if we need the flexibility later we can add it in.

Reply:
ok sounds good. as long as y'all have a way of specifying everything you need when running it, I'm satistfied